.github/workflows/extra_tests.yml

```diff
@@ -118,6 +118,11 @@ jobs:
             -name target -prune \
             -or -name '*.o' -prune \
             -or -print | xargs tar --no-recursion --append -f $TARFILE
+          # Above command skips the whole target directories, but we
+          # need .so outputs. Add those explicitly:
+          find shadow \
+            -path '*/target/*.so' \
+            | xargs tar --no-recursion --append -f $TARFILE
```

Comment:
I think this should be scoped to just `build/src/target`? That way we don't accidentally include a target directory like `src/target` which may exist if we ran `rustfmt --check` or something.

Reply:
I don't think it'd actually hurt much other than to make the artifact a bit bigger. I changed this to explicitly grab just the shim .so, though.